src/pudl_archiver/zenodo/api_client.py

```diff
@@ -178,6 +189,10 @@ async def create_deposition(self, data_source_id: str):
             # Ignore content type
             return Deposition(**await response.json(content_type=None))
 
+    def changed(self):
```

Comment:
Return type annotations aren't great in this module right now, but I feel like one would be useful here

Reply:
Def! Will add.